Dockerfiles/entrypoints/tldw-app-first-run.sh

```
@@ -30,6 +33,86 @@ is_invalid_key() {
+            if "\n" in value:
+                print(f"[entrypoint] Refusing env value with newline for key {key}", file=sys.stderr)
+                sys.exit(1)
+            print(f"{key}={value}")
+except Exception as exc:
+    print(f"[entrypoint] Failed to parse env file {path}: {exc}", file=sys.stderr)
+    sys.exit(1)
+PY
+  then
+    while IFS= read -r assignment || [ -n "$assignment" ]; do
+      [ -n "$assignment" ] || continue
+      export "$assignment"
```

Shellcheck (0.11.0)

[warning] This does not export 'assignment'. Remove $/${} for that, or use ${var?} to quiet.

(SC2163)